utils/chsmack.c

```diff
@@ -196,9 +205,75 @@ static void print_file(const char *path)
 	if (rc > 0) {
 		printf(" transmute=\"%s\"", label);
 		free(label);
+		n = 1;
 	}
 
-	printf("\n");
+	printf(n ? "\n" : ": No smack property found\n");
+}
+
+static void explore(const char *path, void (*fun)(const char*), int follow)
+{
+	struct stat st;
+	int rc;
+	char *file;
+	size_t last, length, l;
```

Comment:
One more non-descriptive variable. This name is also bad because it looks like 1.